dynamo/models/cube.js

```diff
@@ -162,16 +163,31 @@ const hydrate = async (cube) => {
   }
 
   cube.owner = await User.getById(cube.owner);
+  cube.collaborators = await User.batchGet((cube.collaborators || []).filter((id) => id));
   cube.image = getImageData(cube.imageName);
-
+  
```

Comment:
Try to make sure you run `npm run prettier` before each commit (or set up autoformatting in your IDE) so that we don't add any new formatting errors. (There are plenty of them as is, but that's being worked on)

Reply:
I will definitely do that before making this no longer a draft